README.md

```diff
@@ -58,6 +58,14 @@ If you want to set host to a specific IP address for example `172.17.12.1` run `
 
 *To view visit http://localhost:8000/demos/<name>*
 
+## Guidelines for CSS Development
+
+- For issues created in Core that will affect a component in PF-React, a follow up issue must be created in PF-React once the Pull Request is merged.
+- The UX/Visual designer and CSS developer of a component/enhancement should be involved in the review process throughout the life cycle of that component/enhancement.
```

Comment:
Should we be more explicit here and say "once the component/enhancement is complete it should receive sign off from a visual designer who can then update the master sketch file with any need changes"?

Reply:
yep updated the last point with your suggestion and removed `The UX/Visual designer and CSS developer of a component/enhancement should be involved in the review process throughout the life cycle of that component/enhancement.` as I think it becomes redundant.